frontend/appflowy_flutter/lib/plugins/database_view/widgets/row/cells/url_cell/url_cell.dart

```diff
@@ -122,60 +127,61 @@ class _GridURLCellState extends GridCellState<GridURLCell> {
       value: _cellBloc,
       child: BlocBuilder<URLCellBloc, URLCellState>(
         builder: (context, state) {
-          final richText = Padding(
-            padding: GridSize.cellContentInsets,
-            child: FlowyText.medium(
-              state.content,
-              color: Theme.of(context).colorScheme.primary,
-              decoration: TextDecoration.underline,
+          final urlEditor = Padding(
+            padding: EdgeInsets.only(
+              left: GridSize.cellContentInsets.left,
+              right: GridSize.cellContentInsets.right,
             ),
-          );
-
-          return AppFlowyPopover(
-            margin: EdgeInsets.zero,
-            controller: _popoverController,
-            constraints: BoxConstraints.loose(const Size(300, 160)),
-            direction: PopoverDirection.bottomWithLeftAligned,
-            triggerActions: PopoverTriggerFlags.none,
-            offset: const Offset(0, 8),
-            child: SizedBox.expand(
-              child: GestureDetector(
-                child: Align(alignment: Alignment.centerLeft, child: richText),
-                onTap: () async {
-                  final url = context.read<URLCellBloc>().state.url;
-                  await _openUrlOrEdit(url);
-                },
+            child: TextField(
+              controller: _controller,
+              focusNode: _focusNode,
+              maxLines: 1,
+              style: (widget.cellStyle?.textStyle ??
+                      Theme.of(context).textTheme.bodyMedium)
+                  ?.copyWith(
+                color: Theme.of(context).colorScheme.primary,
```

Comment:
Are there UI mockups for this?

Reply:
I used regular Text Cell's UI for this but added underlining as a decoration and took the link color from the previous implementation

Reply:
There isn't a UI mockup for this feature. We can create an design request for this feature later, but it would take some time.